ads-with-uniregistry.rst

```diff
@@ -0,0 +1,73 @@
+
+Today we're excited to announce an important sponsorship partner in our `Ethical Advertising`_ campaign: `Uniregistry`_.  Our goal with our ethical advertising program is to provide important funding for open source, and show that it can be done ethically.
+
+Domain registration was identified early as a natural partner to our program, because it sits in the stack of necessary infrastructure for all of us that work on making the internet. We wanted the right partner, because historically we feel that domain registration companies left room for improvement in the customer experience. This is post covers a few of the criteria we used to reach the conclusion to partner with Uniregistry.
+
+To not bury the lede,
+we're offering 25% off first year registrations and/or transfers into Uniregistry,
+using the code ``DOCS25``.
+We hope that you'll enjoy getting started with the service,
+and find it as useful.
+`Sign up`_ now to get started.
+
+Here are some of the reasons that Uniregistry is a good domain host:
+
+Usability
+---------
+
+Uniregistry makes every interaction point simple, minimal and as sharp looking as possible. This pays off in a good complete experience that should delight you. It doesn’t suck.
+
+Domain registration starts with search, and their clutter-free, fast-loading search gets the job done across hundreds of TLDs worldwide. They make everything you might need to do possible within one or two clicks, instead of 10.
+
+They also have native apps on iOS and Android that work well, and make it easy to grab a new domain, manage your existing names, or transfer in and out at your leisure.
+
+Privacy and Security
+--------------------
+
+Uniregistry offers free WHOIS Privacy on every domain. No nickle-and-diming to protect your personal information. In recent years, those of you who have your phone numbers and emails on domain registration have come to experience being heckled by automated phone calls and spam. Uniregistry takes this off the table with no extra cost to you.  Uniregistry’s competitors charge between $3.99 and $7.99 for what should be your right, not a paid privilege.
+
+Two step verification is built right into the product as well. They are the registrar for people who own one domain or thousands, so they had to build in security and tools to put you in complete control. Since they have handled the use case of the most demanding domain owners in the industry, you get the same benefits even if you just buy one.
+
+Price
+-----
+
+Our analysis showed that most domain registrars which provide good pricing are constantly up-selling and playing renewal games that cost money.  Uniregistry offer transparent, honest pricing on your most valuable assets. They have built a business like we would, they have never charged its customers for WHOIS privacy, they believe that freedom from unwelcome solicitation or online harassment is a fundamental right, not an opportunity for an upsell.
```

Comment:
Only complaint here is "Our analysis" sounds awfully official :) perhaps something more casual here.